.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Release and Publish
+
+on:
+  push:
+    branches:
+      - main
+      - next
+
+jobs:
```

Comment:
We should have a test job in here so we do not publish if tests break:

```
 test:
  runs-on: ubuntu-latest
  steps:
  - name: Checkout
    uses: actions/checkout@v3
  - name: Setup Node.js
    uses: actions/setup-node@v3
    with:
      node-version: 16.x
  - name: npm install
    run: |
      npm install
  - name: npm lint
    run: |
      npm run lint
  - name: npm test
    run: |
      npm test
```

Reply:
> We should have a test job in here so we do not publish if tests break:
> 
> ```
>  test:
>   runs-on: ubuntu-latest
>   steps:
>   - name: Checkout
>     uses: actions/checkout@v3
>   - name: Setup Node.js
>     uses: actions/setup-node@v3
>     with:
>       node-version: 16.x
>   - name: npm install
>     run: |
>       npm install
>   - name: npm lint
>     run: |
>       npm run lint
>   - name: npm test
>     run: |
>       npm test
> ```

@trygve-lie agreed! There are no tests in this repo yet though. Should I still add it now?

Reply:
Yeah. We should write some tests but that is kinda out of the scope for this PR I think. Though; there should be a test command in npm in this PR at least. If it just echo's I think its fine in this PR.